net-p2p/createtorrent/Manifest

```diff
@@ -1 +1,4 @@
 DIST createtorrent-1.1.4.tar.gz 80605 BLAKE2B e17d5ce6e319c8bc2b5eabbc72c3c632eefaba6edbaa2aadcb0be82a556472d6779d0d0d9feb23b5f9dd58eb5104cc448986f452fbb56139f1ea342c721de41e SHA512 d15c6d38e90a2941c578396fafaaa1778d5ce597681e0db89189b5c6719e8b69202dd6bb9dd2c279ef939eef8a74dd8e7ca3f0c224a5a4eaa277c3fd9543c24f
+EBUILD createtorrent-1.1.4-r1.ebuild 650 BLAKE2B 9576b252d56b9f055ea3af88941e5dd41b00fb9182e45b1b9030588288aa0cf41244538fa231d1ed69794935d7415951723ce0592402136088610b72f3bdbdf9 SHA512 6c3be6bb5443b29594c7f24867b187da3b7f6119fd3224859ab381173b36ce619a2f5736ba4884580ce21aadaa96da803de89872a9e37ba86552b90c0e298258
+EBUILD createtorrent-1.1.4.ebuild 635 BLAKE2B 2313c835167bcdb379276ce847dd30ed266074eb566ebfdeec409adc472623e3e7d288c6ee6eefe13d2109c6a144d4692ff209977501ec092102ba90b56e9049 SHA512 94f2e744b8917cef8a33113493097b33443a1d7ec0842370eb8db0ef423f1e99898c2d6e6971c34c3563103a7b4eb5d9e739358432235d0ff086fe9347edfbce
+MISC metadata.xml 166 BLAKE2B c254f1fb642881aba57637be14fb0a89b10384f91a128feaec3a8c870d76efc2cbacb92caccc0dee2dd19a5ac5eaf8643080dafa05c4e2ac96a68568927e5afd SHA512 a56648c974a1d14dd4c18237532773c72057a13ab90c58b5da04f185e3c12a8bd8d5c21fb06053507f31766291a82dc7d87b34cd65fd94cfe2af7295c813ef84
```

Comment:
Why are you adding these? You should use repoman

Reply:
I understand that but I have not been able to figure out how to use repoman to commit the new file and to properly generate the pull request.  Can you point me to the right docs?

Reply:
https://wiki.gentoo.org/wiki/Gentoo_GitHub

Reply:
> And how does github and the portage tree used to test and develop new ebuild interact? In particular, if I do a clone of gentoo/gentoo, what is the best way to debug changes to it in a gentoo environment?

You can test your changes with ebuild command. For example
ebuild createtorrent-1.1.4-r1.ebuild manifest clean merge